pkg/commands/drift/drift.go

```diff
@@ -325,7 +325,9 @@ func (d *IAMDriftDetector) URI(i *assetinventory.AssetIAM) string {
 			resourceName = p.Name
 		}
 		return fmt.Sprintf("/organizations/%s/projects/%s/%s/%s", d.organizationID, resourceName, role, i.Member)
-	} else {
+	} else if i.ResourceType == assetinventory.Organization {
```

Comment:
nit: It would be slightly clearer if we use `switch`

```go
switch i.ResourceType {
case assetinventory.Folder:
  //...
case assetinventory.Project:
  //...
case assetinventory.Organization:
  //...
default:
  return fmt.Sprintf("unknownParent:/organizations/%s/%s/%s/%s/%s", d.organizationID, i.ResourceType, i.ResourceID, role, i.Member)
}
```

Reply:
Done